src/quo/components/wallet/account_overview/schema.cljs

```diff
@@ -0,0 +1,20 @@
+(ns quo.components.wallet.account-overview.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:account-name {:optional true} [:maybe :string]]
+      [:currency-change {:optional true} [:maybe :string]]
+      [:current-value {:optional true} [:maybe :string]]
```

Comment:
Shouldn't this be `:number` ?

Reply:
Good point. `:string` is too loose to represent numbers. For instance, if the value is a string because it's an encoded big integer, then it would be better to at least create an alias to represent this, e.g. `:schema.common/raw-bigint`. I'm not sure if that's the case here, but the `:string` schema is also a good example of a type that doesn't help me figure out how to deal with it.

In the future, we could also consider only passing actual numbers throughout the app, even if some of them are representations of big numbers and need to be decoded as such. I did this in the past and it can bring sanity to the whole system. But we would need to make sure all numbers are decoded when written to the app-db.